### ISSUE TITLE
[joomla.org] 

### ISSUE THREAD

**Issue by cysurfer** [joomla/joomla-websites#1243]
I am unable to locate a language switcher on the docs.joomla.org site so how do I change my default view to English on the site? Parts of the site are displayed in Spanish but I am an English speaker so it is difficult to navigate the site or even search for the information I am looking for.
<img width="1403" alt="screen shot 2018-10-20 at 11 52 21 am" src="https://user-images.githubusercontent.com/41320928/47257705-3cb0c700-d45f-11e8-8baa-0761d6c429e6.png">

**Comment by Sandra97** [comment]
JDocs displays the pages according to the language settings of your browser.

**Comment by Hutchy68** [comment]
Or create an account on docs and you can choose your language. Just make sure you add some text to your user page. Unused accounts are dropped eventually.

**Comment by cysurfer** [comment]
My browser is set to English. If you see the image I shared, some of the site is shown in English and some of it is shown in Spanish, especially some of the menu items. It was difficult to create an account for this reason but I did create an account and once I did that, I was able to set preferences to English which changed all of the displays to English. The first time I accessed the Joomla Docs, my browser was set to English (and it still is) but I was physically sitting in Mexico so I'm guess the site automatically set it to Spanish. It's just crazy that there is no way to set it to English without creating an account first.
